server/certidp/certidp.go

```diff
@@ -53,7 +53,13 @@ var (
 )
 
 func GetStatusAssertionStr(sa int) string {
-	return StatusAssertionValToStr[StatusAssertionIntToVal[sa]]
+	v, ok := StatusAssertionIntToVal[sa]
+	if !ok {
+		// set unknown as fallback
```

Comment:
It would be worth expanding this comment here to explain that this is necessary because `Good` is the default `iota` value (https://pkg.go.dev/golang.org/x/crypto/ocsp#pkg-constants) and that we don't want to default to `Good` if there's no map entry. 

For that matter, it's probably worth noting that in `MarshalJSON` and `UnmarshalJSON` too.

Reply:
Okay, I have added comments to the functions.